packages/flutter/lib/src/material/app.dart

```diff
@@ -53,7 +54,7 @@ class MaterialApp extends StatefulWidget {
     this.theme,
     this.home,
     this.routes: const <String, WidgetBuilder>{},
-    this.initialRoute,
+    this.initialRoute = Navigator.defaultRouteName,
```

Comment:
style nit: we use ':' for defaults rather than '=' (for historical reasons at this point, but we want the code to be consistent)